.github/release-drafter.yml

```diff
@@ -0,0 +1,24 @@
+name-template: 'v$NEXT_PATCH_VERSION'
```

Comment:
We can safely remove this file from the PR. This is a workflow for building Meshery (not applicable in this repo). 

Our direct aim is the ISSUE_TEMPLATES folder, and configuration of a couple bots.

Reply:
the file has been removed @leecalcote